apps/odyssey-stats/src/widget/mini-chart.tsx

```diff
@@ -91,7 +109,6 @@ const MiniChart = ( { siteId, quantity = 7, gmtOffset, odysseyStatsBaseUrl } ) =
 						isPlaceholder={ isEmptyChart }
 					>
 						<StatsEmptyState
-							headingText=""
```

Comment:
We must still pass the empty string to `StatsEmptyState` to prevent rendering the [default heading text](https://github.com/Automattic/wp-calypso/blob/update/migrate-odyssey-typescript-2/client/my-sites/stats/stats-empty-state/index.tsx#L16).

Reply:
Good catch Thanks I missed this when splitting off the pr.